### DISCUSSION TITLE
You are implementing react-datePicker using Hook Form.

### DISCUSSION THREAD

**Discussion by thunder-fury** [react-hook-form/react-hook-form#6948] [General]
Hello👋

You are implementing DatePicker using Hook Form.
There is no problem to display the calendar.

I want to specify the required item calendar, but it is experiencing difficulty to do it.
Can you tell me if there is a good way

Stop in code below😭

```tsx
<Controller
    control={control}
    name={name}
    render={({ field: { onChange, value } }) => (
       <ReactDatePicker
          className={datePicker}
          dateFormat={`yyyy-MM-dd`}
          placeholderText={`placeholderText`}
          selected={value as Date}
          onChange={onChange}
       />
    )}
 />
```

**Comment by fahadsohail482** [comment]
@thunder-fury can you share code sandbox link here.

**Reply by thunder-fury** [reply]
@fahadsohail482 

Here will.
Please confirm.🙏

https://codesandbox.io/s/ancient-architecture-tttoq?file=/pages/index.js
